src/services/storage-options/restApiService.js

```diff
@@ -17,19 +23,22 @@ export default class RestApiService extends StorageInterface {
       session: state.session,
     };
     const credential = state.auth.credentials?.yatt[0];
+    const cookies = document.cookie;
+    console.log(cookies);
     let authHeader;
     authHeader = `Bearer ${credential.accessToken}`;
```

Comment:
there is already a function for building the header in `YattIntegrationHelpers.js` use that instead of copying the logic